site_scons/site_tools/doneapi.py

```diff
@@ -18,22 +18,19 @@ class DetectCompiler():
 
     def __init__(self):
         root = '/opt/intel/oneapi/compiler/latest'
-        binp = os.path.join(root, 'linux', 'bin')
-        libp = os.path.join(root, 'linux', 'lib')
-        include = os.path.join(root, 'linux', 'include')
-        binarch = os.path.join(binp, 'intel64')
-        libarch = os.path.join(root, 'linux', 'compiler', 'lib', 'intel64_lin')
+        binp = os.path.join(root, 'bin')
+        libp = os.path.join(root, 'lib')
+        include = os.path.join(root, 'include')
         icx = os.path.join(binp, 'icx')
         self.map = {}
         sys.stdout.flush()
-        for path in [root, binp, libp, binarch, libarch, include, icx]:
+        for path in [root, binp, libp, include, icx]:
             if not os.path.exists(path):
+                print("oneapi compiler: %s doesn't exist" % path)
```

Comment:
This introduced a pylint failure
> consider-using-f-string, Formatting a regular string which could be an f-string
```suggestion
                print(f"oneapi compiler: {path} doesn't exist")
```

Reply:
Done.